daemon/daemon.py

```diff
@@ -300,8 +300,12 @@ async def persistent_sessions(app):
   conn_big_thumbs = aiohttp.TCPConnector(ssl=sslcontext, limit=8, family=family)
   app['SESSION_BIG_THUMBS'] = session_big_thumbs = aiohttp.ClientSession(connector=conn_big_thumbs, trust_env=trust_env)
 
+  timeout = aiohttp.ClientTimeout(total=60*60) # 1 hour
```

Comment:
Adding increased timeouts and later setting it for asset downloads and asset uploads.